Tests/UnitTests/Plugins/Cuda/Seeding2/TestHostCuts.hpp

```diff
@@ -46,6 +46,6 @@ class TestHostCuts : public Acts::IExperimentCuts<TestSpacePoint> {
   cutPerMiddleSP(
       std::vector<typename Acts::CandidatesForMiddleSp<
           const Acts::InternalSpacePoint<TestSpacePoint>>::value_type>
-          seedCandidates) const final override;
```

Comment:
Why was this removed, it does not seem to be related to the PR?

Reply:
clang-tidy suggestion (which I ran on the entire suite).